packages/flutter/lib/src/material/list_tile_theme.dart

```diff
@@ -462,6 +477,12 @@ class ListTileTheme extends InheritedTheme {
   /// [ListTileThemeData.enableFeedback] property instead.
   bool? get enableFeedback => _data != null ? _data.enableFeedback : _enableFeedback;
 
+  /// Overrides the default value of [ListTile.enableFeedback].
```

Comment:
Looks like this comment was copied but not updated.

Reply:
resolved, see https://github.com/flutter/flutter/pull/146630/commits/550d9ed65a70d9afadbe3255d27d1035dbc9b688